### PR TITLE
New resource reservation status (TAM-160)

### PR DESCRIPTION
**Pull request by urshala**
Following new reservation status are added:

    * vapaa {hours} asti/ free until {hour}: Show this status if the
      resource is free now but is reserved later for the day. E.g. If 
      the resource is free now and can be reserved but it has one 
      reservation at 16:00. Then render `free until 16:00`
      
    * tarkista saatavuus/ External reservation: If the resource should
      be reserved outside the varaamo. These kind of resources can
       be reserved by manually calling the phone numbers...
       
    * suljettu tilapaisesti/ temporarily closed: If the resource is
      not reservable.

### CODE REVIEW

File: app/i18n/messages/sv.json [hunk 1]
```diff
@@ -349,6 +349,9 @@
   "ResourceAvailability.availableAt": "Ledig kl. {time}",
   "ResourceAvailability.availableTime": "Ledig {hours} {hours, plural, one {timme} other {timmar}}",
   "ResourceAvailability.closed": "Stängt",
+  "ResourceAvailability.closedTemporarily": "Closed temporarily __swedish__",
```
Comment (wiltso) [inline app/i18n/messages/sv.json]:
```suggestion
  "ResourceAvailability.closedTemporarily": "Tillfälligt stängt",
```

File: app/i18n/messages/sv.json [hunk 2]
```diff
@@ -349,6 +349,9 @@
...
+  "ResourceAvailability.freeUntil": "free until {time} __swedish__",
```
Comment (wiltso) [inline app/i18n/messages/sv.json]: If with free, you mean it doesn't cost anything then change it to this, in other cases can you specify how in what way you mean it's free.
```suggestion
  "ResourceAvailability.freeUntil": "gratis till {time}",
```

File: app/i18n/messages/sv.json [hunk 3]
```diff
@@ -349,6 +349,9 @@
...
+  "ResourceAvailability.externalReservation": "Reservation elsewhere __swedish__",
```
Comment (wiltso) [inline app/i18n/messages/sv.json]:
```suggestion
  "ResourceAvailability.externalReservation": "Bokning någon annanstans",
```

**Review by wiltso — COMMENTED** [review]
Please add the translations

File: app/i18n/messages/sv.json (hunk 2 above)
Comment (urshala) [reply app/i18n/messages/sv.json]: Free here means more like `available`

Comment (wiltso) [reply app/i18n/messages/sv.json]: Then it should be "tillgänglig till {time}"

**Review by wiltso — CHANGES_REQUESTED** [review]

File: app/i18n/messages/sv.json [hunk 4]
```diff
@@ -349,6 +349,9 @@
   "ResourceAvailability.availableAt": "Ledig kl. {time}",
   "ResourceAvailability.availableTime": "Ledig {hours} {hours, plural, one {timme} other {timmar}}",
   "ResourceAvailability.closed": "Stängt",
+  "ResourceAvailability.closedTemporarily": "Tillfälligt stängt",
+  "ResourceAvailability.freeUntil": "tillgänglig till {time}",
```
Comment (wiltso) [inline app/i18n/messages/sv.json]: Should it be a start with a capital letter?

**Review by wiltso — COMMENTED** [review]
Looks good

**Pull request merged**